src/lib/resolveMigrationDocumentData.ts

```diff
@@ -48,13 +48,19 @@ export async function resolveMigrationContentRelationship(
 		) {
 			return {
 				...(await resolveMigrationContentRelationship(relation.id)),
-				// TODO: Remove when link text PR is merged
-				// @ts-expect-error - Future-proofing for link text
-				text: relation.text,
+				...("text" in relation && relation.text && { text: relation.text }),
```

Comment:
could you explain why this error wasn't triggered when we merged link text, and why we ended up using `in` to solve the issue?